lib/widget/dropdown.dart

```diff
@@ -247,15 +247,15 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
                 item.icon != null
                     ? iconframework.Icon.fromModel(item.icon!)
                     : SizedBox(
-                  width: widget._controller.size.toDouble(),
+                  width: widget._controller.gap.toDouble(),
```

Comment:
please see above (this is not the gap)

Reply:
I used that sizedbox because icon is existing in list but there's no icon in that object so it can give gap between them and the next size box is giving space between icon and text that’s why it’s necessary to use both to keep it aligned